Servers/deploy-ha-exchange-on-clc.md

```diff
@@ -0,0 +1,127 @@
+{{{
+  "title": "Deploy HA Exchange on CenturyLink Cloud",
+  "date": "4-2-2015",
+  "author": "Mark Turpin",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Exchange 2013 on CenturyLink Cloud – High Availability and Automatic Failover
+
+Exchange 2013 offers excellent high availability and disaster recovery for enterprise messaging. With CenturyLink Cloud’s Global Data Center footprint, it is easy to deploy a globally scaled and highly available Exchange architecture.
+
+### Prerequisites
+
+Exchange 2013 requires an Active Directory environment.  AD can easily be automated through the [CLC create server wizard](../servers/creating-a-new-enterprise-cloud-server.md) or through [deploying an AD blueprint](../blueprints/deploy-microsoft-windows-2012-active-directory-domain-services.md).  For the purpose of this article, an AD forest exists already, or has been deployed from a CLC blueprint.  A domain controller will be required in each datacenter (and minimal resources should suffice, depending on environment).  This article also assumes you have Domain Admin permissions (and Exchange Organization Admin).  Load balancers and additional mailbox servers could be added to this design for larger organizations.
+
+### Design
+
+Our design for a basic Exchange 2013 DR/HA environment will involve 3 datacenters and 3 servers.  For example, UT1 is our primary/active Exchange datacenter; IL1 is our secondary/passive Exchange datacenter, and NY1 will hold the file share witness (FSW) for the Exchange Database Availability Group (DAG).
+
+(../images/deploy-ha-exchange-on-clc-1.png)
```

Comment:
All images are broken. The markdown format for images is `![Alt text](../images/deploy-ha-exchange-on-clc-1.png)`

Fix these up and we'll get this merged